environment/src/test/java/com/sequenceiq/environment/environment/v1/converter/EnvironmentApiConverterTest.java

```diff
@@ -174,35 +172,6 @@ void testInitEditDto() {
         verify(networkRequestToDtoConverter).convert(request.getNetwork());
     }
 
-    @ParameterizedTest
-    @ValueSource(booleans = { false, true })
-    void testAzureSingleRgEnabledWithEmptyAzureRequest(Boolean azureSingleResourceGroupDeploymentEnabled) {
```

Comment:
May I ask, why was this deleted?

Reply:
Because now, even if entitlement for single RG exists, resources will not be deployed into a single automatically, only if the name of the RG and SINGLE usage is provided. From now on cloudbreak needs a name I think.

Reply:
After personal communication we agreed to reintroduce the test. I also added another one.